ssr-tests/pages/index.tsx

```diff
@@ -92,6 +92,8 @@ import {
   NxTree,
   NxVulnerabilityDetails,
   NxWarningAlert,
+  NxFileUpload,
+  NxStatefulFileUpload,
```

Comment:
Just to confirm, all the new built components should be added to this file, correct?

Reply:
> Just to confirm, all the new built components should be added to this file, correct?

Yes. This file is what verifies that the components can be rendered by a server-side rendering framework

Reply:
Got it, thanks.